pallets/runtime/tests/src/staking/mod.rs

```diff
@@ -5231,16 +5231,25 @@ fn should_add_permissioned_validators() {
         let acc_10 = 10;
         let acc_20 = 20;
 
-        assert_ok!(Staking::add_permissioned_validator(
+        provide_did_to_user(10);
+        provide_did_to_user(20);
+
+        assert_ok!(Staking::add_permissioned_validator_entity(
             frame_system::RawOrigin::Root.into(),
-            acc_10.clone()
+            Identity::get_identity(&acc_10).unwrap()
         ));
-        assert_ok!(Staking::add_permissioned_validator(
+        assert_ok!(Staking::add_permissioned_validator_entity(
             frame_system::RawOrigin::Root.into(),
-            acc_20.clone()
+            Identity::get_identity(&acc_20).unwrap()
         ));
-        assert_eq!(Staking::permissioned_validators(acc_10), true);
-        assert_eq!(Staking::permissioned_validators(acc_20), true);
+        assert_eq!(
+            Staking::permissioned_entities(Identity::get_identity(&acc_10).unwrap()),
+            true
+        );
```

Comment:
Let's make a function out of this, taking the AccountId + the true/false RHS of assert_eq and refactor below.